### ISSUE TITLE
WindowProperty:FrameAndDivider window and frame projections do not correctly affect heat loss

### ISSUE THREAD

**Issue by sullivj1** [NREL/EnergyPlus#10296]
Issue overview
--------------
I was doing modelling assumptions and sensitivity tests on window frame inputs using the WindowProperty:FrameAndDivider inputs, and I found two issues with the heat loss:
1. Frame inside and outside projection have no impact on frame convective heat transfer, though they do affect shading.
2. Divider inside and outside projection do affect frame convective heat transfer, however their impact appears to be almost twice what would be expected.

To isolate the heat loss effects and remove the confounding effects of frame geometry on solar gains, I surrounded the entire model in a shading box:
![image](https://github.com/NREL/EnergyPlus/assets/149975523/85fc8e20-7811-4a1a-af6e-970062ee3c1c)

Observed heating use for this model (run in Wellington, New Zealand) was:
70mm aluminium frame	                        3,286.93kWh
+30mm inside and outside projection	3,286.93kWh
100mm frame for same surface area	3,407.51kWh

140mm (70x2) dividers	                        3,261.66kWh
+30mm inside and outside projection	3,530.96kWh
+30mm frame added	                        3,378.86kWh

As be seen, adding inside and outside frame projections in the absence of any shading effects has no effect on the overall heating use, which should not be the case given that the InputOutputReference states that they should affect convection from the frame.

Looking at dividers, with the same surface area they result in basically the same numbers as a frame. However, here adding divider projections significantly increases the heat loss, much more than the (roughly) equivalent area of frame. Examining the frame heat transfer directly, the average heat loss with 140mm dividers was -3.1W. Adding 70mm inside and outside projections, which should in theory double the exposed surface area, almost quadruples the average heat loss to -11.2W – almost twice what we would expect. Given that window frame heat loss should essentially be a simple function of surface area (all else being equal), this does not seem correct.

### Details
Some additional details for this issue (if relevant):
 - Platform: Windows 11
 - Energyplus version 22.1.0

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Defect file added ([WindowFrameTests.zip](https://github.com/NREL/EnergyPlus/files/13261059/WindowFrameTests.zip))
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by joseph-robertson** [comment]
I believe that I can reproduce these issues. At least the general trends. Using one of the attached defect files, StudioTest5_frame70_nosun.idf, I ran four variations of this base file:
* (A) as-is, no changes
* (B) set Frame Outside/Inside Projection = 0.03
* (C) set Number of Horizontal/Vertical Dividers = 1
* (D) set Number of Horizontal/Vertical Dividers = 1 and Divider Outside/Inside Projection = 0.03

Comparing (A) to (B), I see no change in results (heating use, heat loss, etc).
Comparing (C) to (D), I see an increase in (D) relative (C).

I think we'd expect both, or neither, to show an increase. (But probably both since the docs say that the projection fields are "used to calculate ... convection from frame" and "used to calculate ... convection from divider".)

One related thing to clarify: aren't the dividers effectively 280mm (140x2) in the defect files? It looks like Divider Width = 0.14 and both Number of Horizontal/Vertical Dividers = 1. Not sure if this would account for the comment about quadrupling the average heat loss.

@sullivj1

**Comment by sullivj1** [comment]
@joseph-robertson 
> One related thing to clarify: aren't the dividers effectively 280mm (140x2) in the defect files? It looks like Divider Width = 0.14 and both Number of Horizontal/Vertical Dividers = 1. Not sure if this would account for the comment about quadrupling the average heat loss.

They're being set to be (approximately) equivalent to the 70mm wide frames. Basically the horizontal divider is equivalent to the top and bottom frame segment, while the vertical divider is equivalent to the two side frame segments. See diagram:
<img width="548" height="225" alt="Image" src="https://github.com/user-attachments/assets/96bec45d-4b45-461b-a124-a0896039ce0c" />

**Comment by joseph-robertson** [comment]
I think I figured out at least the first part of this. See https://github.com/NatLabRockies/EnergyPlus/pull/11589.

Basically the calculation/assignment of both `SurfWinProjCorrFrOut` and `SurfWinProjCorrFrIn` was only happening if dividers were _also_ present.

**Comment by joseph-robertson** [comment]
> [@joseph-robertson](https://github.com/joseph-robertson)
> 
> > One related thing to clarify: aren't the dividers effectively 280mm (140x2) in the defect files? It looks like Divider Width = 0.14 and both Number of Horizontal/Vertical Dividers = 1. Not sure if this would account for the comment about quadrupling the average heat loss.
> 
> They're being set to be (approximately) equivalent to the 70mm wide frames. Basically the horizontal divider is equivalent to the top and bottom frame segment, while the vertical divider is equivalent to the two side frame segments. See diagram: <img alt="Image" width="548" height="225" src="https://private-user-images.githubusercontent.com/149975523/592142425-96bec45d-4b45-461b-a124-a0896039ce0c.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Nzg3ODkzOTcsIm5iZiI6MTc3ODc4OTA5NywicGF0aCI6Ii8xNDk5NzU1MjMvNTkyMTQyNDI1LTk2YmVjNDVkLTRiNDUtNDYxYi1hMTI0LWEwODk2MDM5Y2UwYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwNTE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDUxNFQyMDA0NTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iYjllYTQ4NWE5MzQzNDljYzBhMGUwNWMwMDQ2ZGY5Mjg5NDE0ZGExZmJlYmFhNjBjZTgzYTdjYjhmZmEwMjk3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZyZXNwb25zZS1jb250ZW50LXR5cGU9aW1hZ2UlMkZwbmcifQ.BfHSRxvr2rHTvVTpPw9OoE-YG3XraPgm8iBK185TJuU">

@sullivj1 Thinking about this some more... if you are adding 70mm projections to both the inside and outside, isn't the exposed surface area being doubled on both the inside and outside?

**Comment by sullivj1** [comment]
@joseph-robertson right, but the inside and outside area would (in a typical 2-dimensional plane) be the same. Imagine instead that we just have a flat plane rather than a 3d projection - doubling its area would result in both the inside and outside face doubling in area. Correspondingly, the heat loss also doubles.

We can illustrate this basic principle in THERM. Here I modelled a steel block and changed the surface conditions so that it is either just the flat width of the “frame” (a) or has exposed inside and outside projections (b) (Figure 58). As we would expect, the heat loss is proportional to the surface area. Doubling the exposed surface area via 3D ‘projections’ results in a doubling of the heat loss, not a quadrupling like EnergyPlus’s divider projection inputs are suggesting. 

<img width="847" height="450" alt="Image" src="https://github.com/user-attachments/assets/26de5ada-62db-4070-8878-59f6255ae8d1" />
Figure 58. Rough illustration of the effect of frame geometry on heat loss in THERM using a rectangular steel projection. 

It gets more complicated if you have asymmetrical projections. Iirc, external surface area has a greater effect on heat loss than internal though don't quote me on that.

~~~~~~~~
As a note though if you're looking to fix this I'm not sure it's necessarily helpful for WindowFrameAndDivider projections to increase the heat loss of the frames? The issue is that the basic workflow for getting the frame and IGU properties for a model is to export an idf fragment from Window 7. Frame U-values, however, are essentially a "flattened" description of them as if their surface area was the projected plane area. This is how aluminium frames have physically impossible U-values of U7+ that would not be able to be modelled in EnergyPlus (surface resistances limited U-values in practice to ~5.8) - their real 3d surface area is higher than implied by the projected frame width. 

Frame projections actually affecting heat loss could potentially make them less usable since it means that people cannot use standard frame U-values as inputs to Window 7 and EnergyPlus when doing so. I'm not sure what the implications would be for workflows and the general accessibility of Energyplus's detailed window constructions. Would it mean that anyone wanting to include geometry and projections for their shading effects needs to import a detailed frame model from THERM?

(the window frames also have deeper problems than this, see issue [here](https://github.com/NatLabRockies/EnergyPlus/issues/10445))

**Comment by joseph-robertson** [comment]
The fix in https://github.com/NatLabRockies/EnergyPlus/pull/11589 addresses the first item in the description at the top of this issue.

I'm not totally sure how to address the second item, so I'm going to unassign myself.
